java/arcs/core/storage/keys/RamDiskStorageKey.kt

```diff
@@ -32,8 +32,10 @@ data class RamDiskStorageKey(private val unique: String) : StorageKey(RAMDISK_DR
         private val RAMDISK_STORAGE_KEY_PATTERN = "^(.*)\$".toRegex()
 
         init {
-            // When RamDiskStorageKey is imported, this will register its parser with the storage
-            // key parsers.
+            // When RamDiskStorageKey is used for the first time, this will register its parser 
```

Comment:
Ray is looking into exactly what type of 'use' is needed to run this init block.